pkg/virtual/workspaces/registry/rest.go

```diff
@@ -257,8 +257,55 @@ func (s *REST) authorizeOrgForUser(ctx context.Context, orgClusterName logicalcl
 	return nil
 }
 
+func (s *REST) authorizeForUser(ctx context.Context, orgClusterName logicalcluster.Name, user user.Info, verb string, resourceName string) error {
+	// Root org access is implicit for every user. For non-root orgs, we need to check for
+	// verb permissions against the clusterworkspaces/workspace sub-resource.
+	if orgClusterName == tenancyv1alpha1.RootCluster || sets.NewString(user.GetGroups()...).Has("system:masters") {
```

Comment:
please use the https://github.com/kubernetes/kubernetes/blob/ab4aa182a5808c9e61413fadc051d99d6572b0ab/staging/src/k8s.io/apiserver/pkg/authentication/user/user.go#L71 constant

Reply:
Fixed in commit cd4ae7e7873e38fa9d93b0513c7bd763f070894b